macos/FluentUI/Badge/BadgeView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+// MARK: - BadgeView
+
+@available(*, deprecated, renamed: "BadgeView")
+public typealias MSBadgeView = BadgeView
+
+@objc(MSFBadgeView)
+open class BadgeView: NSView {
+	/// Initializes a Fluent UI Badge View with the provided title, default style, and small size
+	/// - Parameters:
+	///   - title: String displayed in the badge
+	@objc public convenience init(title: String) {
+		self.init(title: title, style: .default, size: .small)
+	}
+
+	public required init?(coder aDecoder: NSCoder) {
+		preconditionFailure("init(coder:) has not been implemented")
+	}
+
+	/// Initializes a Fluent UI Badge View with the provided title, style, and size
+	/// - Parameters:
+	///   - title: String displayed in the badge
+	///   - style: The BadgeViewStyle, currenly only default is supported
```

Comment:
why even have this param if default is the only thing that is supported

Reply:
same with size

Reply:
I thought it'd make it easier to add the other styles and sizes later if we wanted to.

Reply:
is that the direction that design team is going? otherwise, it is bit confusing we have that api that dosn't do anything. once we have those variation it make sense to have this convenience method added to public.